tests/NotificationTest.php

```diff
@@ -419,10 +420,41 @@ public function it_can_set_notification_sound()
 
         $notification = $notification->toArray();
 
-        $this->assertCount(4, $notification);
+        $this->assertCount(6, $notification);
         $this->assertSame($sound, $notification['sound']);
     }
 
+    /**
+     * @test
+     */
+    public function it_allows_setting_play_sound_to_false()
+    {
+
+        $notification = Notification::make()->playSound(false);
+
+        $this->assertInstanceOf(Notification::class, $notification);
+
+        $notification = $notification->toArray();
+
+        $this->assertCount(5, $notification);
+        $this->assertFalse($notification['play_sound']);
+    }
+
+        /**
```

Comment:
Code indentation looks a bit odd here.

Reply:
Apologies, I have made an update now to fix this